extensions/binding/org.eclipse.smarthome.binding.fsinternetradio/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="fsinternetradio"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Sample Thing Type -->
+    <thing-type id="radio">
+        <label>Internet Radio</label>
+        <description>An internet radio device based on the Frontier Silicon chipset.</description>
+
+        <channels>
+            <channel id="power" typeId="power"/>
+            <channel id="mode" typeId="mode"/>
+            <channel id="volume" typeId="volume"/>
+            <channel id="mute" typeId="mute"/>
+            <channel id="play-info-name" typeId="play-info-name"/>
+            <channel id="play-info-text" typeId="play-info-text"/>
+            <channel id="preset" typeId="preset"/>
+        </channels>
+
+        <config-description>
+            <parameter name="ip" type="text" required="true">
+                <label>Internet Radio Address</label>
+                <description>The IP address (name or numeric) of the internet radio.</description>
+            </parameter>
+            <parameter name="port" type="integer" required="true">
+                <label>Internet Radio Port</label>
+                <description>The port of the internet radio (default: 80).</description>
+                <default>80</default>
+            </parameter>
+            <parameter name="pin" type="text" required="true">
+                <label>Internet Radio Pin</label>
+                <description>The PIN configured in the internet radio (default: 1234).</description>
+                <default>1234</default>
+            </parameter>
+            <parameter name="refresh" type="integer">
+                <label>Refresh interval</label>
+                <description>Specifies the refresh interval in seconds.</description>
+                <default>60</default>
+            </parameter>
+        </config-description>
+    </thing-type>
+
+    <channel-type id="power">
+        <item-type>Switch</item-type>
+        <label>Internet Radio Power</label>
+        <description>Switch the radio on or off.</description>
+        <category>Switch</category>
+    </channel-type>
+    <channel-type id="preset">
+        <item-type>Number</item-type>
+        <label>Internet Radio Preset</label>
+        <description>Preset radio stations configured in the radio.</description>
+    </channel-type>
+    <channel-type id="volume">
+        <item-type>Number</item-type>
```

Comment:
It would be great if you could define the volume as a Percent type as UIs can then render sliders for it.
You can do the mapping from 0-32 to 0-100 within your binding code then.